### ISSUE TITLE
Is there a hotkey for "Copy image to clipboard" button in image editor?

### ISSUE THREAD

**Issue by lferrao** [ShareX/ShareX#3192]
If yes, it would be nice to add it to the tooltip to avoid questions like these :)

**Comment by iamsolarplexus** [comment]
Indeed.  I'm coming from Greenshot where I'm used to copying what I've just grabbed, but only maybe (i.e., I don't want that to be the default action.  I may need to edit the image first or I may abandon it)

**Comment by NameLess--** [comment]
If the image editor you are talking about is "Greenshot Image Editor" (which ShareX uses), the shortcut is Shift+Control+C

**Comment by lferrao** [comment]
It doesn't work. I'm talking about this editor right here called ShareX Image Editor:

![2018-03-22 1](https://user-images.githubusercontent.com/6348118/37760142-19d3748a-2db6-11e8-9802-0418ac143378.png)

**Comment by lferrao** [comment]
@Jaex  What version contains these shortcuts? Cause I have [v12.1.1](https://github.com/ShareX/ShareX/releases/tag/v12.1.1) and it doesn't seem to work.

**Comment by Jaex** [comment]
Dev build: https://getsharex.com/docs/dev-builds

**Comment by lferrao** [comment]
Hi, will this ever make it into the Windows Store build?

**Comment by lferrao** [comment]
Just saw that it did, thanks!

**Comment by ekhcentrics** [comment]
Can we please make the keybinding for copy configurable?  Not having it be the standard control-C on windows is very frustating.
(This is on 20.0.4)
